### PR TITLE
Sequencer/ftx other rejection reason

### PR DESCRIPTION
**Pull request by Filter94**
This PR implements issue(s) #2089

### Checklist

* [ ] I wrote new tests for my new core changes.
* [ ] I have successfully ran tests, style checker and build against my new changes locally.
* [ ] I have informed the team of any breaking changes if there are any.

---

> [!NOTE]
> **Medium Risk**
> Changes forced-transaction pool retry/status behavior for unrecognized rejections, which can affect forced-tx ordering and observability; tests mitigate but production behavior may change for edge-case rejection reasons.
> 
> **Overview**
> Improves forced-transaction handling for *unknown/other* rejection reasons by explicitly treating them as retryable (no inclusion status recorded), clarifying logging in `LineaForcedTransactionPool`.
> 
> Adds acceptance-test coverage for this scenario by constructing a forced tx that overflows a module limit (BLS `MAP_FP_TO_G1`) and asserting it blocks subsequent forced txs while returning `null` inclusion status; also factors shared BLS overflow calldata + expected error into `BlsMapFpToG1OverflowSetup` and updates `EstimateGasModuleLimitOverflowLimitlessTest` to reuse it. Updates test tx construction to use a fixed `CONTRACT_CALL_GAS_LIMIT` (300k) to match enforced FTX limits.
> 
> <sup>Written by [Cursor Bugbot](https://cursor.com/dashboard?tab=bugbot) for commit 684ffa3d30e31cac94e5e2e8a3733ef4ddd70a58. This will update automatically on new commits. Configure [here](https://cursor.com/dashboard?tab=bugbot).</sup>

### CODE REVIEW

File: besu-plugins/linea-sequencer/acceptance-tests/src/test/kotlin/linea/plugin/acc/test/rpc/linea/BlsMapFpToG1OverflowSetup.kt [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+ * SPDX-License-Identifier: MIT OR Apache-2.0
+ */
+package linea.plugin.acc.test.rpc.linea
+
+import linea.plugin.acc.test.tests.web3j.generated.BLS12_MAP_FP_TO_G1
+import org.apache.tuweni.bytes.Bytes
+
+/**
+ * Shared setup for tests that simulate PRECOMPILE_BLS_MAP_FP_TO_G1_EFFECTIVE_CALLS overflow.
+ * Factor out of EstimateGasModuleLimitOverflowLimitlessTest and reused in ForcedTransactionTest.
```
[the remark this answers is not in the document]
Comment (Filter94) [reply besu-plugins/linea-sequencer/acceptance-tests/src/test/kotlin/linea/plugin/acc/test/rpc/linea/BlsMapFpToG1OverflowSetup.kt]: That is very insightful, thank you!

File: besu-plugins/linea-sequencer/acceptance-tests/src/test/kotlin/linea/plugin/acc/test/rpc/linea/ForcedTransactionTest.kt [hunk 2]
```diff
@@ -503,6 +503,40 @@ class ForcedTransactionTest : AbstractForcedTransactionTest() {
...
+  @Test
+  fun forcedTransactionOverflowingOtherLimitBlocksFollowingForcedTransactions() {
+    val bls12MapFPtoG1 = deployBLS12_MAP_FP_TO_G1()
+    val sender = accounts.secondaryBenefactor
+    val recipient = accounts.primaryBenefactor
+
+    val overflowCallData = BlsMapFpToG1OverflowSetup.encodeOverflowCall(bls12MapFPtoG1)
+    val rawTxOverflow =
+      createSignedContractCall(sender, bls12MapFPtoG1.contractAddress, overflowCallData, 0)
+    val rawTxValid = createSignedTransfer(sender, recipient, 0)
```
[the remark this answers is not in the document]
Comment (Filter94) [reply besu-plugins/linea-sequencer/acceptance-tests/src/test/kotlin/linea/plugin/acc/test/rpc/linea/ForcedTransactionTest.kt:515]: This is correct, because if TX1 is rejected, TX2 has chances to be executed, but without a conclusive rejection status that shouldn't be possible
